### PR TITLE
ingested naacl 2021

### PR DESCRIPTION
**Pull request by xinru1414**

### CODE REVIEW

**Comment by xinru1414** [comment]
@mjpost `ingest.py` prompted to add a line in `data/sigs/mai.yaml`. However we do not currently have this file.

**Comment by mjpost** [comment]
One TODO item: add all the workshops to `data/yaml/joint.yaml`, so they'll appear jointly on [the event page](https://aclanthology.org/previews/ingestion-naacl-2021/events/naacl-2021/).

File: data/xml/2021.naacl.xml [hunk 1]
```diff
@@ -0,0 +1,6017 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2021.naacl">
+  <volume id="srw" ingest-date="2021-05-21">
```
Comment (mjpost) [inline data/xml/2021.naacl.xml:3]: Please reorder the volumes:
* main
* demos
* srw
* tutorials
* industry

**Comment by mjpost** [comment]
> @mjpost `ingest.py` prompted to add a line in `data/sigs/mai.yaml`. However we do not currently have this file.

Which workshop mentioned this (grep through the `meta` files)? It could be a new SIG or it could be a mistake.

**Comment by xinru1414** [comment]
> > @mjpost `ingest.py` prompted to add a line in `data/sigs/mai.yaml`. However we do not currently have this file.
> 
> Which workshop mentioned this (grep through the `meta` files)? It could be a new SIG or it could be a mistake.

It's the `2021.maiworkshop-1 # Proceedings of the Third Workshop on Multimodal Artificial Intelligence`. I couldn't find this workshop in any of the current `.yaml` files we have. The meta file also suggests :`sig mai`. I think it's a new SIG, and will proceed to add a new `.yaml` file

**Comment by mjpost** [comment]
This doesn't sound like a SIG. I'd just assume it was a mistake, and don't add anything. If they reach out to us later, we can add it.

**Comment by xinru1414** [comment]
> One TODO item: add all the workshops to `data/yaml/joint.yaml`, so they'll appear jointly on [the event page](https://aclanthology.org/previews/ingestion-naacl-2021/events/naacl-2021/).

Done

**Review by mjpost — APPROVED** [review]
LGTM. Good to merge at [2021-05-24 0:00 UTC+14](https://time.is/UTC+14).

**Pull request merged**